tests/test_security_system.py

```diff
@@ -30,6 +38,21 @@ def test_face_detection(self):
 
         # Check if faces were detected in the processed image
         self.assertTrue(len(processed_image) > 0)
+        
+        # Check if the image is not entirely black
+        self.assertTrue(np.mean(processed_image) > 0)  
```

Comment:
I don't think we need to assert the image being black after we've performed face detection.